stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -38,27 +38,30 @@ import kotlinx.coroutines.withContext
 import kotlinx.parcelize.Parcelize
 
 internal class DefaultFlowController internal constructor(
-    private val activity: ComponentActivity,
+    private val viewModelStoreOwner: ViewModelStoreOwner,
```

Comment:
Do all of these need to be val?

Reply:
I am surprised this is not picked up as a ktlint warning... it shows up in Android Studio as a yellow line.